### PR TITLE
[REM] test_event_full,test_crm_full: perf tests

### PR DESCRIPTION
**Pull request by Julien00859**
Remove performance tests from both tef and tcf.

Too many PR are blocked due to broken assertQueryCount, either there are too many queries, either there are too few.

It is too much work to run all tests three times only to update a comment with the final count (module alone + community + enterprise).

It is too hard to keep track of the hundreds queries to determine those that moved, those that are missing and those that are new between two branches. We have to apply tons of string-replace and sorts just to help some diff tools (e.g. meld) into showing what changed.

Basically, except a few people, nobody care to do the investigation work and just increase the query count (without changing the comments).

We tried for one year, now it is time to let those test go.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
You may also deactivate those tests, as some people use them when developing features. But I agree they are tedious to update, and nobody cares. So in short, would just skip them on runbot, and keep the code for people that want to work with it (aka, people in our team).

Bisous.

**Comment by tde-banana-odoo** [comment]
cc @Xavier-Do

**Comment by thiryr** [comment]
:+1: Query count is also 'weirdly' unreliable with caching and all (hence `@warmup`, I guess). I confirm tracing these is not fun nor very useful, for my 2 cents.

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Comment by tde-banana-odoo** [comment]
Hehe, blocked by a random error, that's funny.
